Xamarin.Forms.Core/Picker.cs

```diff
@@ -35,7 +35,11 @@ public class Picker : View, IFontElement, ITextElement, IElementConfiguration<Pi
 		public static readonly BindableProperty FontSizeProperty = FontElement.FontSizeProperty;
 
 		public static readonly BindableProperty FontAttributesProperty = FontElement.FontAttributesProperty;
-		
+
+		public static readonly BindableProperty PlaceholderProperty = PlaceholderElement.PlaceholderProperty;
```

Comment:
the `Picker` class should implement `IPlaceholderElement` so it can be selected by CSS

Reply:
Fixed